1611(docs): improves clarity for profile (#1631)
hyperledger/besu-docs: docs/public-networks/how-to/use-configuration-file/profile.md

---
title: Use a profile 
sidebar_position: 2
---

Load a profile using the [`--profile` CLI option](../../reference/cli/options.md#profile).

> Profiles are optionally used extend Besu's configuration. They may be used together with the [boilerplate configuration](defaults.md) to apply sensible defaults. Besu's pre-configured profiles optimize for the following supported use cases:

Reviewer: I think rewrite for active voice and also use "might" instead of "may".

```suggestion
You can optionally use profiles to extend Besu's configuration. Combine profiles with the [boilerplate configuration](defaults.md) to apply sensible defaults. Besu's pre-configured profiles optimize for the following supported use cases:
```
https://learn.microsoft.com/en-us/style-guide/grammar/verbs#active-and-passive-voice

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/can-may

What are "sensible defaults"? Is there another way to say this?

```bash
besu --profile=minimalist_staker
``` -->

## Staker profile

`--profile=staker` is optimized for stakers who want to maximize their hardware value while also serving full sets of data to their peers. See the
[staker profile on GitHub](https://github.com/hyperledger/besu/blob/8b64023a121ea996ef60e4b7e2299c5807683f90/config/src/main/resources/profiles/staker.toml)
for the custom settings.

<!-- IMO this is all redundancy -- fully supported by the linked page above

To use the staker profile, run Besu with [`--profile`](../../reference/cli/options.md#profile) set to `staker`:

```bash
besu --profile=staker
``` -->

## Enterprise/Private profile

`--profile=private` / `--profile=enterprise` supports private network operators and enterprises by handling specific use cases that apply to private network operators. See the [enterprise/private profile on GitHub](https://github.com/hyperledger/besu/blob/8b64023a121ea996ef60e4b7e2299c5807683f90/config/src/main/resources/profiles/enterprise-private.toml)
for the custom settings.

<!-- IMO this is all redundancy -- fully supported by the linked page above

> To use the enterprise/private profile, run Besu with

Reviewer: I think this link should be included and not commented out.

Author: OK -- going to keep single source but duplicate links to that source
